compliance/compliance.go

```diff
@@ -2013,3 +2018,51 @@ func ForwardReferencesDisallowed(c *fluent.GRIBIClient, t testing.TB, _ ...TestO
 			AsResult(),
 		chk.IgnoreOperationID())
 }
+
+// AddNHGReferencingToUknownNH tests a case where a nexthop group references a nexthop IP that does not Resolve.
+func AddNHGReferencingToUknownNH(c *fluent.GRIBIClient, wantACK fluent.ProgrammingResult, t testing.TB, _ ...TestOpt) {
```

Comment:
This test should probably be in `openconfig/featureprofiles` rather than a compliance test here -- since it depends on the DUT having this IP unresolved.

If we want to have something here that references an unresolved IP we'd need to extend our compliance tests (and interaction with lemming) to cover that. I'd recommend adding this as a test case in featureprofiles instead.

The same is true for the down port case.